providers/fips/fipsprov.c

```diff
@@ -325,7 +325,7 @@ void ERR_put_error(int lib, int func, int reason, const char *file, int line)
      * won't work for third party providers that have their own error mechanisms,
      * so we'll need to come up with something else for them.
      */
```

Comment:
This comment seems stale now.

Reply:
Good point, although we'll still need to work on newer error reporting functionality that somehow involves either `OSSL_PROVIDER` or `OPENSSL_CTX`, at least if we're going to move to library context tied error stacks...  that will need a discussion, methinks.

But anyhow, consider the comment removed, happens in the new few minutes.

Reply:
...  or changed.  I ended up changing it rather than removing it...